src/systems/bit-media-frames.js

```diff
@@ -215,9 +229,32 @@ function mediaTypesOf(world, entities) {
   return mask;
 }
 
+const takeOwnershipOnTimeout = new Map();
 const heldQuery = defineQuery([Held]);
 // const droppedQuery = exitQuery(heldQuery);
 export function mediaFramesSystem(world) {
+  enteredMediaFramesQuery(world).forEach(eid => {
+    if (Networked.owner[eid] === APP.getSid("reticulum")) {
+      takeOwnershipOnTimeout.set(
```

Comment:
Also for this timer pattern in general, like we have discussed a few times in the past already.